ui/hooks/gasFeeInput/useGasFeeInputs.js

```diff
@@ -102,10 +102,13 @@ export function useGasFeeInputs(
   });
 
   const [estimateUsed, setEstimateUsed] = useState(() => {
```

Comment:
[nit] I feel like 'estimateTypeUsed' or 'estimateLevelUsed' would be a clearer name for this. Not blocking obviously.